home-exercises-05-2022/src/io/codelex/arithmetic/practice/HourlyCalculator.java

```diff
@@ -0,0 +1,27 @@
+package io.codelex.arithmetic.practice;
+
+public class HourlyCalculator {
+    static void hourlyEmployeesPayCalculator(double basePay, int hoursWorked) {
+        boolean error = false;
+        if (basePay < 8.00) {
+            System.out.println("Error: Minimum wage is $8.00 an hour.");
+            error = true;
+        }
+        if (hoursWorked > 60) {
+            System.out.println("Error: Hours are greater than 60.");
+            error = true;
```

Comment:
Ja base pay ir normāls jebšu lielāks par 8 un hours worked ir par daudz, tad šo mainīgo uzstādīt šeit nav jēgas. Jo ja izpildāšs šis IF bloks, tad else if to pārbaudi nekad neveiks.